cluster/addons/addon-manager/README.md

```diff
@@ -1,25 +1,25 @@
 ### Addon-manager
 
-addon-manager manages two classes of addons with given template files.
+addon-manager manages two classes of addons with given template files in `$ADDON_PATH`.
```

Comment:
If we are going to introduce $ADDON_PATH earlier here, maybe we should also move its default setting note "(default `/etc/kubernetes/addons/`)" here?

Reply:
Done.